addons/website/controllers/main.py

```diff
@@ -236,10 +236,24 @@ def create_sitemap(url, content):
 
         return request.make_response(content, [('Content-Type', mimetype)])
 
-    @http.route('/website/info', type='http', auth="public", website=True, sitemap=True)
+    def sitemap_website_info(env, rule, qs):
+        website = env['website'].get_current_website()
+        if not (
+            website.viewref('website.website_info', False).active
+            and website.viewref('website.show_website_info', False).active
+        ):
+            # avoid 404 or blank page in sitemap
+            return False
+
+        if not qs or qs.lower() in '/website/info':
+            yield {'loc': '/website/info'}
+
+    @http.route('/website/info', type='http', auth="public", website=True, sitemap=sitemap_website_info)
```

Comment:
> Note that we probably don't want to keep indexing this page, as it is
a technical link which doesn't bring any value to our clients to be
included in their sitemap.
It might have been useful for Odoo to perform some google search / stats
or something like that.
But since this is a stable fix, the behavior is kept as is as much as
possible.

Mmh, I think I would rather even been less conservative and just put `sitemap=False` :open_mouth: Do you plan to do that in master? But as you wish in stable, what you did looks fine :wink:

Reply:
Yes in master I was going to do it with a triple check from internal: maybe it's there for some reason I don't know.
Only thing I am sure of in stable was this part :p

Reply:
https://discord.com/channels/@me/811500922252296203/1106122011811516468